src/otg/v2g.py

```diff
@@ -36,16 +38,24 @@ def run(self: V2GStep) -> None:
         # Filter gene index by approved biotypes to define V2G gene universe
         gene_index_filtered = GeneIndex.from_parquet(
             self.session, self.gene_index_path
-        ).filter_by_biotypes(self.approved_biotypes)
+        ).filter_by_biotypes(list(self.approved_biotypes))
 
         vi = VariantIndex.from_parquet(self.session, self.variant_index_path).persist()
         va = VariantAnnotation.from_parquet(self.session, self.variant_annotation_path)
         vep_consequences = self.session.spark.read.csv(
             self.vep_consequences_path, sep="\t", header=True
+        ).select(
+            f.element_at(f.split("Accession", r"/"), -1).alias(
+                "variantFunctionalConsequenceId"
+            ),
+            f.col("Term").alias("label"),
+            f.col("v2g_score").cast("double").alias("score"),
         )
 
         # Variant annotation reduced to the variant index to define V2G variant universe
-        va_slimmed = va.filter_by_variant_df(vi.df, ["id", "chromosome"]).persist()
+        va_slimmed = va.filter_by_variant_df(
+            vi.df, ["variantId", "chromosome"]
```

Comment:
Why do you need to specify the columns you join the two dataframes? Isn't it always the same? What other usecases are there?

Reply:
I don't think we use this function in any other way. It is more flexible to select the joining cols, but hard coding them to variantid and chromosome shouldn't cause problems. Thanks

Reply:
Thanks.